docs/src/modules/components/DemoToolbar.js

```diff
@@ -717,7 +717,7 @@ export default function DemoToolbar(props) {
 DemoToolbar.propTypes = {
   codeOpen: PropTypes.bool.isRequired,
   codeVariant: PropTypes.string.isRequired,
-  copyButtonOnClick: PropTypes.object.isRequired,
+  copyButtonOnClick: PropTypes.func.isRequired,
```

Comment:
Nice! I was about to open a PR to fix this prop type 👏

Reply:
Some time I open my dev tools and spot warning errors 😁